packages/ckeditor5-ckbox/docs/features/ckbox.md

```diff
@@ -46,66 +32,194 @@ Image files are inserted into the content as images that you can drag around and
+					toolbar: ['imageUpload', 'ckbox', 'ckboxImageEdit', /* ... */ ], // Depending on your preference.
+					ckbox: {
+						// Configuration.
+					}
+				} )
+				.then( /* ... */ )
+				.catch( /* ... */ );
+		</script>
+	</body>
+</html>
+```
 
-<code-switcher>
-```js
-import { ClassicEditor, Image, ImageUpload, PictureEditing, CKBox, CKBoxImageEdit, CloudServices } from 'ckeditor5';
+### Method 2: Installation from npm
 
-ClassicEditor
-	.create( document.querySelector( '#editor' ), {
-		licenseKey: '<YOUR_LICENSE_KEY>',
-		plugins: [ Image, PictureEditing, ImageUpload, CloudServices, CKBox, CKBoxImageEdit, /* ... */ ],
-		toolbar: [ 'ckbox', 'ckboxImageEdit', /* ... */ ], // Depending on your preference.
-		ckbox: {
-			// Configuration.
-		}
-	} )
-	.then( /* ... */ )
-	.catch( /* ... */ );
-```
-</code-switcher>
+For applications using modern bundling tools (such as Webpack or Vite) and package managers (npm, yarn, pnpm), CKBox components should be installed as dependencies and imported into your CKEditor 5 build.
 
-Further in the document, the dependency plugins will be omitted in code listings for clarity.
+<info-box note>
+	CKBox is a premium feature. Using the npm/yarn distribution channel (Self-hosted) requires having an appropriate commercial plan or a Custom plan that permits the use of the editor via npm or a ZIP package.
 
-### Adding the image editing button to the image toolbar
+	You can [sign up](https://portal.ckeditor.com/checkout?plan=free) for the CKEditor Premium Features 14-day free trial to test the feature. If you already have a license, you should log in to your Customer Portal Dashboard to access feature settings and credentials.
+</info-box>
 
-To invoke the CKBox image editor straight from the image contextual toolbar (available at right-click in an image), it needs to be added to the toolbar configuration.
+First, install the necessary dependencies:
 
-{@img assets/img/toolbar-items.png 402 An extended contextual toolbar.}
+```bash
+npm install ckeditor5 ckbox
+```
 
-The snippet below shows an example image contextual toolbar configuration.
+Once the packages are installed, you can import the necessary modules directly into your application code and use them within your application:
 
 ```js
+import { ClassicEditor, Image, ImageUpload, PictureEditing, CKBox, CKBoxImageEdit, CloudServices } from 'ckeditor5';
+import * as ckbox from 'ckbox';
+import 'ckbox/dist/styles/ckbox.css';
+
 ClassicEditor
 	.create( document.querySelector( '#editor' ), {
-		// ... Other configuration options ...
-		image: {
-			toolbar: [ 'toggleImageCaption', 'imageTextAlternative', 'ckboxImageEdit' ]
+		licenseKey: '<YOUR_LICENSE_KEY>',
+		plugins: [ LinkEditing, Image, PictureEditing, ImageUpload, CloudServices, CKBox, CKBoxImageEdit, /* ... */ ],
+		toolbar: [ 'imageUpload', 'ckbox', 'ckboxImageEdit', /* ... */ ], // Depending on your preference.
+		ckbox: {
+			// Configuration.
```

Comment:
If I remember correctly, the `tokenUrl` property is required, so maybe we could add it? When someone would like to copy-paste this snippet will get an error about the missing property